mongo/integration/mtest/mongotest.go

```diff
@@ -586,6 +586,11 @@ func (t *T) TrackFailPoint(fpName string) {
 
 // ClearFailPoints disables all previously set failpoints for this test.
 func (t *T) ClearFailPoints() {
+	if t.clientOpts != nil && t.clientOpts.AutoEncryptionOptions != nil && len(t.failPointNames) > 0 {
+		t.Logf("configureFailPoint is not supported for auto encryption, skipping ClearFailPoints()")
+		t.failPointNames = t.failPointNames[:0]
```

Comment:
The added spec tests include both `AutoEncryptionOptions` and `"configureFailPoint"`, which seems to suggest that `"configureFailPoint"` is being run successfully during the test (to enable the failpoint). Does disabling failpoints behave differently than enabling failpoints when `AutoEncryptionOptions` are set?

Reply:
Yes, clearing fail point fails for "mongocrypt error 1: command not supported for auto encryption: configureFailPoint".